pytest-embedded-qemu/pytest_embedded_qemu/qemu.py

```diff
@@ -55,11 +95,28 @@ def __init__(
 
         qemu_prog_path = qemu_prog_path or self.qemu_prog_name
 
+        self.current_qemu_executable_path = qemu_prog_path
+        self.image_path = image_path
+        self.efuse_path = qemu_efuse_path
+
         if qemu_cli_args:
             qemu_cli_args = qemu_cli_args.strip('"').strip("'")
         qemu_cli_args = shlex.split(qemu_cli_args or self.qemu_default_args)
         qemu_extra_args = shlex.split(qemu_extra_args or '')
 
+        if self.efuse_path:
+            logging.debug('The eFuse file will be saved to: %s', self.efuse_path)
+            with open(self.efuse_path, 'wb') as f:
+                f.write(QEMU_DEFAULT_EFUSE[self.app.target])
+            qemu_extra_args += [
+                '-global',
+                f'driver={self.app.target}.gpio,property=strap_mode,value=0x08',
```

Comment:
`strap_mode` is dependent on the chip target, please search for `strap_mode` in IDF's qemu_ext.py.